pyproject.toml

```diff
@@ -81,3 +83,23 @@ find = {}  # Scanning implicit namespaces is active by default
 
 [tool.setuptools.dynamic]
 version = {attr = "pyperformance.__version__"}
+
+[tool.mypy]
+python_version = "3.7"
```

Comment:
It's generally a good idea to run mypy on the lowest Python version you support; this means it will flag if you accidentally use a feature that only exists in a newer Python version